todo_list_class.rb

```diff
@@ -0,0 +1,63 @@
+require_relative 'todo_item_class.rb'
+
+class ToDoList
+	def initialize(name = "TODO")
+		@name = name
+		@list = Hash.new #maps name of item to item object
+	end
+
+	def name
```

Comment:
`attr_reader` might be a helpful tool here. Let me know if you need more hints than that.